src/Theme/PageTemplate.elm

```diff
@@ -28,7 +23,7 @@ view model content =
     div [ css [ pageWrapperStyle ] ]
         [ globalStyles
         , div [ css [ pageStyle ] ]
-            [ main_ [ css [ mainStyle ] ]
+            [ main_ []
                 [ button [ onClick LanguageChangeRequested ] [ text (t ChangeLanguage) ]
                 , div []
```

Comment:
We might want to move the `pageWrapperStyle` `div` here and possibly the container inside that also if it is common to all pages.